src/test/java/net/snowflake/client/jdbc/SnowflakeConnectionV1Test.java

```diff
@@ -198,4 +205,19 @@ public void testMergeProperties() {
     assertThat(result.get("PROP1"), is("value1|value2"));
     assertThat(result.get("PROP2"), is("carrot^"));
   }
+
+  @Test
+  public void testCreateArrayOfIsCaseInsensitive() throws SQLException {
+    SFConnectionHandler mockConnectionHandler = mock(SFConnectionHandler.class);
+    SFBaseSession sfBaseSession = mock(SFBaseSession.class);
+    when(mockConnectionHandler.getSFSession()).thenReturn(sfBaseSession);
+    when(sfBaseSession.checkProperties()).thenReturn(new ArrayList<>());
+    try (SnowflakeConnectionV1 connectionV1 = new SnowflakeConnectionV1(mockConnectionHandler)) {
```

Comment:
why is it a test with mocked connection? can we have integration test showing how it is used from external user perspective?

Reply:
We already have one ([BindingAndInsertingStructuredTypesLatestIT.java#L264](https://github.com/snowflakedb/snowflake-jdbc/blob/a20f2cff732c68b176351d7566f4743e79a65379/src/test/java/net/snowflake/client/jdbc/BindingAndInsertingStructuredTypesLatestIT.java#L264)) so I decided to add a unit test instead as its point is to guard the behaviour of accepting both upper and lower case

Reply:
Personally I would prefer to enrich the existing test then

Reply:
cool, will do